README.md

```diff
@@ -368,5 +382,44 @@ const toast2 = showToast("number 2", configs);
 ```
 
 ### Styling the toast
-
 TODO
+
+## Security and Privacy Considerations
+See [TAG Security / Privacy Self Review](/security-privacy-self-review.md).
+
+## Considered Alternatives
+
```

Comment:
Accessibility section? Even it's also empty.

Reply:
I'm actually working on an accessibility study in another branch, I can add the header in here though